CHANGELOG.md

```diff
@@ -1,17 +1,135 @@
+
+#### New Contributors
+@garretrieger made their first contribution in #110
+
+Full Changelog: v0.6.1...v0.6.2
+
+
+### v0.6.1 (2023-May-3)
+#### Changes since last release:
+
+- [glyphsets]Added Okina in Beyond set by @RosaWagner in #103
+- [nam] Add Kawi by @simoncozens in #106
+- [nam] Add Nag Mundari by @simoncozens in #105
+- [nam] Add Chorasmian glyphset by @simoncozens in #90
+- [nam] Add Braille glyphset by @simoncozens in #91
+- [nam] Add more yi codepoints by @simoncozens in #100
+- [nam] Fill various holes by @simoncozens in #97
+- [nam] Add UCAS Extended A codepoints by @simoncozens in #93
+- [nam] Add IPA extensions by @simoncozens in #94
+- Add missing Unicode 15.0 scripts by @simoncozens in #107
+- [nam] Add all Arabic codepoints by @simoncozens in #92
+- [nam] Add Ottoman Siyaq Numbers by @simoncozens in #108
+
+Full Changelog: v0.6.0...v0.6.1
+
+
+### v0.6.0 (2023-Mar-21)
+#### Changes since previous release:
+
+- Add SignWriting #84
+- Improve coverage test name reporting #89
+- Build subsets.SUBSETS dynamically #95
+- Add test strings (attempt 2) #104
+
+### v0.5.4 (2022-Nov-16)
+#### Changes since last release:
+
+- #83 Add Toto and Tangsa nam files
+
+### v0.5.3 (2022-Oct-7)
+#### Changes since previous release:
+
+- #82: Removed U+20A4 (lira) from GF_Laptin_Plus
+- #81: Replaced Minorities by Beyond
+- #80 #79 #77 #73: adds missing glyphs to subsets
+- #78: Removed A/a caron from Latin_Core
+- #74 #72 #71 #70: Improved Phonetic glyphsets
+- #63: Corrected African glyphset
+- #69 #67: added missing glyphs to subsets (Armenian and Myanmar)
+
+### v0.5.2 (2022-Jun-28)
+#### Changes since previous release:
+
+- missing_glyphsets_in_font: check keys not values #f0a1af223401271dc737f0003f37a67a63006f71
+
+
+### v0.5.1 (2022-Jun-23)
+#### Changes since last release:
+
+- Small changes Latin #60
+- Fixed threshold #62
+
+
+### v0.5.0 (2022-Apr-29)
+#### Changes since previous release:
+
+- Removed WIP from directory name #49
+- Gf glyphset update #50
+- Updated bash script #51
+- Updated Latin's readme.md #52
+- Changed naming scheme, updated database and added translatin dir #53
+- Updated readme.md with instructions to use `glyphsets` tool #54
+- Add get_glyph method and refactor missing_glyphs #55
+
+
+### v0.4.0 (2022-Apr-21)
+#### Changes since previous release:
+
+- Update `glyphsets` #45
+- Update-db: skip glyphs which are non-exportable #48
+- filter-lists: output glyphsapp plist files #47
+- Update db 5d7b788
+
+### v0.3.1 (2022-Apr-11)
+##### Changes since previous release:
+
+- Fix pypi upload `a2907eb` pointing at glyphsets tooling
+
+### v0.3.0 (2022-Apr-11)
+##### Changes since previous release:
+
+- Add `glyphsets` tooling #43 to update the database
+
+### 0.2.1 (2021-Dec-17)
   - Quick bugfix: undefined var `enc_path` (should be `nam_dir` instead!)
 
-## 0.2.0 (2021-Dec-17)
-### Noteworthy code-changes
+### 0.2.0 (2021-Dec-17)
+###### Noteworthy code-changes
```

Comment:
Well... since I mentioned my mild OCD...

Here you're using a 6th level markup (`######`), while in other places, similar entries are using 4 or 5 hashes. Ideally the same level should be used throughout the document, for consistency.

Reply:
fixed